serveradmin/common/static/js/serveradmin.js

```diff
@@ -9,40 +9,30 @@ $(document).ready(function() {
     // everywhere when ever we need it for example when doing long running
     // ajax requests for the servershell search.
     window.spinner = {
-        _running: 0,
+        _startedAt: 0,
```

Comment:
The appropriate initial value for a date would most probably then be `null`.

Reply:
In Javascript it doesn't make any difference:
```
> let now = Date.now()
> now - 0
< 1615449781944
> now - null 
< 1615449781944
```